graphql/resolvers.go

```diff
@@ -4081,23 +4081,33 @@ func New(apiURL string) Config {
 		}
 		return nil, Forbidden.Send(ctx, fmt.Sprintf("user %s does not have permission to access this resolver", user.Username()))
 	}
-	c.Directives.RequireProjectAdmin = func(ctx context.Context, obj interface{}, next graphql.Resolver) (res interface{}, err error) {
-		args, isStringMap := obj.(map[string]interface{})
+	c.Directives.RequireProjectAccess = func(ctx context.Context, obj interface{}, next graphql.Resolver, access ProjectSettingsAccess) (res interface{}, err error) {
+		var permissionLevel evergreen.PermissionLevel
+		if access == ProjectSettingsAccessEdit {
+			permissionLevel = evergreen.ProjectSettingsEdit
```

Comment:
Can we instead just store and pass in value since we don't actually need the description?